src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      data-qa="big"
+      class="form"
+    >
+      <label class="label" for="big">
+        <img
+          src="images/Search.svg"
+          alt="searchIcon"
+          class="iconBig"
+        >
+        <input
```

Comment:
Don't forget to add empty lines between multilane children
```suggestion
        >
        
        <input
```